test/00_operators/OperatorTests.cu

```diff
@@ -1846,6 +1846,47 @@ TYPED_TEST(OperatorTestsNumericAllExecs, OperatorFuncs)
   res = c * c * (c + c) / c + three;
   EXPECT_TRUE(MatXUtils::MatXTypeCompare(tov0(), res, 0.07));
 
+  auto nan = make_tensor<TestType>({});
+  using conversionType = typename matx::detail::value_promote_t<TestType>;  
+  if constexpr(matx::is_complex_v<TestType>) {    
+    nan() = TestType(std::numeric_limits<conversionType>::quiet_NaN());
+  } else {
+    nan() = std::numeric_limits<conversionType>::quiet_NaN();
+  }
+  auto tob = make_tensor<bool>({});
+  // example-begin nan-test-1
```

Comment:
I completely missed this, but you need to add new files into the docs_input directory for these functions. It should be in api/logic/truth. You can copy a file from there and make two new ones for this.